openedx/core/djangoapps/content/course_overviews/migrations/0021_courseoverviewtab_link.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.12 on 2020-04-21 15:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_overviews', '0020_courseoverviewtab_url_slug'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
How many rows are in this table, and will this cause a long running migration to add this column?  It defaults to null, presumably, so I don't _think_ so, but would want to sanity check with someone who's brain has been in MySQL-land more than mine has lately.

Reply:
> also indicates `name` is a problem, right? Does this address that?
> 
> Think I need a bit more background on the fix.

The tab code is receiving a `CourseOverview` object from the database rather than a Course from mongo, and `CourseOverviewTab` stores the tab data, but it previously only stored the basic stuff like `name` and `type`. In this case, the `ExternalLinkTab` expects also to see a `link` attribute, so we need to add that to the model. 

We may run into this problem again if a custom tab expects other data not accounted for. I thought about jamming all of the tab's fields into a `JSONField` on the model, but for now I'm thinking that it's better to be explicit about what we're saving.